content/source/docs/enterprise-beta/api/policies.html.md

```diff
@@ -0,0 +1,256 @@
+      "download":"/api/v2/policies/pol-u3S5p2Uwk21keu1s/download"
+    }
+  }
+}
+```
+
+## Upload a Policy
+
+-> **Note**: This endpoint is not JSON-API enabled therefore the serialization format of the body and the HTTP headers do not follow the JSON-API patterns of other endpoints.
+
+This endpoint is used to upload the policy file to the policy.
+
+
+| Method | Path           |
+| :----- | :------------- |
+| PUT | /organizations/:organization/policies/:policy_id/upload |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - specifies the organization name where the Policy will be defined
+- `:policy_id` (`string: <required>`) - specifies the policy ID to which the file should be saved
+
+### Sample Payload
+
+```plain
+main = rule { true }
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/octet-stream" \
+  --header "Accept: text/plain" \
+  --request POST \
+  --data @payload.json \
+  https://atlas.hashicorp.com/api/v2/organizations/my-organization/policy/pol-u3S5p2Uwk21keu1s/upload
+```
+
+
+## Update a Policy
+
+This endpoint enables you to update a Policy
+
+| Method | Path           |
+| :----- | :------------- |
+| PATCH | /organizations/:organization/policies/:policy_id |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - specifies the organization name where the Policy is defined
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "attributes": {
+      "enforce": [
+        {
+          "path": "my-example-policy.sentinel",
+          "mode": "soft"
+        }
+      ],
+      "name": "my-example-policy",
+    },
+    "type":"policies"
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  --data @payload.json \
+  https://atlas.hashicorp.com/api/v2/organizations/my-organization/policy/pol-u3S5p2Uwk21keu1s
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id":"pol-u3S5p2Uwk21keu1s",
+    "type":"policies",
+    "attributes": {
+      "name":"basic",
+      "enforce": [
+        {
+          "path":"basic.sentinel",
+          "mode":"soft"
+        }
+      ],
+      "updated-at":"2017-10-10T20:58:04.621Z"
+    },
+    "links": {
+      "self":"/api/v2/policies/pol-u3S5p2Uwk21keu1s",
+      "upload":"/api/v2/policies/pol-u3S5p2Uwk21keu1s/upload",
+      "download":"/api/v2/policies/pol-u3S5p2Uwk21keu1s/download"
+    }
+  }
+}
+```
+
+## List Policies
+
+List all the policies for a given organization
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /organizations/:organization/policies |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - specifies the organization name where the Policy is defined
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
```

Comment:
You don't really need this on `GET` requests. No harm, I don't think, but unnecessary.